lib/senders/rust_mailer.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request')
```

Comment:
Let's not call this module `rust_mailer`, Rust is an implementation detail of the service and really of no consequence to anything that happens here. If we reimplemented that API in, say, node or python, all of the code here should stay unchanged.

Reply:
Maybe `email_service` is a good name? Or something else along those lines...

Reply:
Also, not sure if we prefer `_` or `-` in js file names. This repo has both so maybe it doesn't matter.